### ISSUE TITLE
Git reset --hard error on install

### ISSUE THREAD

**Issue by Mx772** [miaoshouai/miaoshouai-assistant#13]
```
GitCommandError: Cmd('git') failed due to: exit code(128) cmdline: git reset --hard cd6f2547fc4b8b59bd7f4ff379307dc5ab1084dd -- stderr: 'fatal: Could not parse object 'cd6f2547fc4b8b59bd7f4ff379307dc5ab1084dd'.'
```

This is due to the submodule requirement, possibly the host. 

I see someone tried to fix it here:
https://github.com/miaoshouai/miaoshouai-assistant/commit/19d1823d451897734e81b21f925123e310001f50

But the issue isn't with the logic git hashes. 

If you can host those files on Github in a separate Repo and then link those as submodules, it should work. 

The hash it's pulling from that submodule is also `cd6f2547fc4b8b59bd7f4ff379307dc5ab1084dd` 

When going to the configured submodule site, I also get a login requirement, or occasionally a 404; both of which are going to prevent users from pulling from there. 
https://e.coding.net/miaoshouai/miaoshou-stable-diffusion-webui/assets.git -> https://e.coding.net/404.html

TL;DR: 
Move those files to this repo, or potentially a new repo to fix the reset --hard issue.

**Comment by miaoshouai** [comment]
Hi Drew, Thanks for commenting on this error. but the strange thing is when you git clone the assets repo separately, it can be done without any problems. so it seems it's not a permission error.

**Comment by miaoshouai** [comment]
ok~ did some fixes. should now resolve both installation and download speed problems

**Comment by miaoshouai** [comment]
if you could try it again and let me know if it makes the fix
